doc/cephfs/cephfs-io-path.rst

```diff
@@ -0,0 +1,29 @@
+ Ceph File System IO Path
+=========================
+
+The Ceph File System (CephFS) stores file data as RADOS objects. File System
+clients access to RADOS directly for reading and writing file data. MDS only
+handle metadata operations.
```

Comment:
What metadata (capabilities/file size) need updated during file writes/reads? What kind of MDS traffic should an application expect?

Please reference and link to file layouts as well.

Reply:
file-layout under cephfs dir is all about settting file layout through xattr.  Nothing related to this.

Reply:
My point is that "... clients access to RADOS directly for reading and writing file data" is not at all specific. We want to explain how the client knows what objects to write (inode number, RADOS namespace, etc.) and which pool those objects should go in. That should include a reference to file layouts.

Reply:
It already references 'Data Striping' section of `Architecture`